src/lib/registration/ui/addon_selection_reregistration_dialog.rb

```diff
@@ -62,6 +64,8 @@ def addon_selected?(addon)
       # empty implementation, allow reregistration of a dependant addon
       # without reregistering its parent
       def reactivate_dependencies
+        # FIXME: how to test this?
```

Comment:
The fixme is to remind us that it is not covered by unit tests and until recently I did not know how to triigger it in the UI, but now I know.